scraping/spiders/poland_rjps.py

```diff
@@ -0,0 +1,119 @@
+from collections import defaultdict
+from typing import List, Dict
+
+import scrapy
+import json
+
+from joblib import Memory
+from config.settings import Settings
+
+memory = Memory(location='cache/poland_rjps')
+settings = Settings()
+DATA_PATH = f'{settings.spider_data_path}/rips'
+COUNTRY_NAME = 'Poland'
+DEFAULT_CATEGORY = "Any Help"
```

Comment:
Should we move `DEFAULT_CATEGORY` to `config.constants`?

Reply:
https://github.com/littlepea/safeway-data/pull/18/commits/1bc78840051c7e5d348e610d3b954a516c53fc0a